src/main/java/com/odysseusinc/arachne/datanode/service/impl/BaseAnalysisServiceImpl.java

```diff
@@ -203,8 +238,9 @@ private Analysis update(Analysis analysis) throws NotExistException {
         } catch (IOException e) {
             LOGGER.warn(Constants.AnalysisMessages.CANT_REMOVE_ANALYSIS_DIR_LOG);
         }
+        final AnalysisResultStatusDTO updatedAnalysisStatus = reEvaluateAnalysisStatus(analysis.getStatus(), resultDir);
         exists.setAnalysisFolder(analysis.getAnalysisFolder());
```

Comment:
If I am not wrong `File analysisFolder = new File(exists.getAnalysisFolder()) ` and `resultDir` is prity much the same dir. So basicly dirictory should not exisist before you run reEvaluateAnalysisStatus.

Reply:
You are wrong, these are two different folders

Reply:
done